dist/jquery.flot.js

```diff
@@ -2015,9 +2014,12 @@ Licensed under the MIT license.
 
         function setRange(axis) {
             var opts = axis.options,
-                min,
-                max,
-                delta;
+                windowWidth = opts.windowSize,
```

Comment:
1. I'm not seeing any documentation for opts.windowSize. Will it be documented in the charting plugin?
2. In case opts.windowSize is being set by a different plugin, I think it's cleaner to use a backup value here (100) instead inside the switch case.
3. Could you please reuse the same name for this variable? windowSize instead of windowWidth?